fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1042,7 +1042,7 @@ ACTOR Future<bool> checkExclusion(Database db,
 						totalKvStoreUsedBytesNonExcluded += used_bytes;
 
 						if (disk_id.empty() || diskLocalities.find(disk_id) == diskLocalities.end()) {
-							totalKvStoreFreeBytes += free_bytes;
+							totalKvStoreAvailableBytes += available_bytes;
```

Comment:
This unfortunately doesn't work if two storage servers are sharing the same disk. The way that we would account for this would be to add the free space from the disk only once and the remaining available space for each storage server, but this is hard to do right now because Redwood chose a different definition for available space that the existing storage engines.

Reply:
There was a related discussion on the last PR that attempted to address this:

https://github.com/apple/foundationdb/pull/8185